hypertrace-view-generator/hypertrace-view-creator/src/main/resources/configs/raw-service-view/application.conf

```diff
@@ -18,7 +18,7 @@ pinot.controllerPort = 9000
 pinot.timeColumn = start_time_millis
 pinot.timeUnit = MILLISECONDS
 # todo: Add Attributes and Metrics after adding map type value
-pinot.dimensionColumns = [tenant_id, span_kind, error_count, exception_count, duration_millis, end_time_millis, api_name, service_name, span_id, trace_id, protocol_name, status_code, service_id, api_id, num_calls, api_discovery_state, space_ids]
+pinot.dimensionColumns = [tenant_id, span_kind, error_count, exception_count, duration_millis, duration_micros, end_time_millis, api_name, service_name, span_id, trace_id, protocol_name, status_code, service_id, api_id, num_calls, api_discovery_state, space_ids]
```

Comment:
why do we need this duration in micro across views, I think as per my understanding, only the span event view needed that (and the backend entity view).

Reply:
@Sunn-y-Arora I think it's better to have this consistently across all views.

Reply:
not required right? raw-service (for eg) view gives service ka overall rsp time, it will never be in micro, so unnecessary change

Reply:
True but why have two different units in two different views? We'll anyway get rid of `duration_millis` across all tables, if extra storage and indexing cost is a concern. This also makes code the query side simpler, as it doesn't have to differentiate b/w the two columns IMO.

Reply:
As pointed out by Prashant, added it in micro across views so that all the views have consistency.